article/templates/article/detail-base.html

```diff
@@ -11,6 +12,23 @@
     {{ article.title }} export guidance - great.gov.uk
 {% endblock %}
 
+{% block sharing_metadata %}
+  <meta name="description" content="" />
+
+  {# Twitter #}
+  <meta name="twitter:card" content="summary">
+  <meta name="twitter:title" content="{{ article.title }}">
+  {# twitter falls back to og for other data not included here #}
+
+  {# Facebook share data #}
+  <meta property="og:locale" content="en_UK" />
+  <meta property="og:site_name" content="great.gov.uk" />
```

Comment:
will hard-coding this cause any problems if we try to share staging or dev urls (for testing purposes) ?

Reply:
as discussed elsewhere: no :)